x/downtime-detector/downtimedetector_module/module.go

```diff
@@ -0,0 +1,129 @@
+package downtimemodule
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/gorilla/mux"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	downtimedetector "github.com/osmosis-labs/osmosis/v13/x/downtime-detector"
+	"github.com/osmosis-labs/osmosis/v13/x/downtime-detector/types"
+	twapcli "github.com/osmosis-labs/osmosis/v13/x/twap/client/cli"
+	"github.com/osmosis-labs/osmosis/v13/x/twap/client/queryproto"
+)
+
+var (
+	_ module.AppModule      = AppModule{}
+	_ module.AppModuleBasic = AppModuleBasic{}
+)
+
+type AppModuleBasic struct{}
+
+func (AppModuleBasic) Name() string { return types.ModuleName }
+
+func (AppModuleBasic) RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
+}
+
+func (AppModuleBasic) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
+	return cdc.MustMarshalJSON(types.DefaultGenesis())
+}
+
+// ValidateGenesis performs genesis state validation for the gamm module.
```

Comment:
```suggestion
// ValidateGenesis performs genesis state validation for the downtime-detector module.
```

Reply:
im just going to delete these comments instead, they're generally boilerplate